src/server.ts

```diff
@@ -0,0 +1,62 @@
+import * as express from 'express';
+import * as socketIo from 'socket.io';
+import * as http from 'http';
+import { config } from './utils/config';
+import * as bodyParser from 'body-parser';
+import { AppRouter } from './router';
+import { userErrorHandler, serverErrorHandler, unknownErrorHandler } from './utils/errors/handler';
+import { logger } from './utils/logger/logger';
+import { ConfigurationSocket } from './sockets/configuration.socket';
```

Comment:
re-organize the imports so that the external packages are first and then the interna imports. (the linter should yell at you so install it if you haven't already)

Reply:
fix that